3lvia/kunde-elvia-web
# Feature/omega 192 han port tabs

OMEGA-X

## What is the reason for this change?

There are tabs in the new design. The HAN page didn't use them until now.

![image](https://user-images.githubusercontent.com/20183313/94681626-13c02e80-0324-11eb-9dde-804ae16dd3bf.png)
![image](https://user-images.githubusercontent.com/20183313/94681782-479b5400-0324-11eb-83b9-efdac1fd22a6.png)
![image](https://user-images.githubusercontent.com/20183313/94681843-67cb1300-0324-11eb-8c26-32dc4001e445.png)

### What is the expected result?

HAN page shows tabs, and there is a tab for going to the HAN page
![image](https://user-images.githubusercontent.com/20183313/94682073-b2e52600-0324-11eb-99a2-896b838b4b9e.png)

Back link is gone from the bottom of the HAN page
![image](https://user-images.githubusercontent.com/20183313/94682096-be385180-0324-11eb-8145-154120e16859.png)

Section with link to HAN page is removed from meter point page, and meter point details list is moved to where this box was before. If the meter is an old meter, box for inputing readings is shown below invoice table.
![image](https://user-images.githubusercontent.com/20183313/94682577-72d27300-0325-11eb-9d5f-87abbb510c69.png)

The HAN tab is only shown if the user has at least one active meter with a HAN port (AMS).

## How is it solved?

I don't really understand this question. It is solved by making the things shown above...

## Any changes/updates that may affect others?

|                       | Changed |
| --------------------- | ------- |
| Unit tests            |       |
| Dependencies          |         |
| Environment variables |         |
| Build scripts         |         |

## How can it be tested and verified?

- [ ] Log in to the customer portal and see that things are as described above
  - Use Ali Jensen for a customer with HAN ports
  - Use Randi Rystad for a customer with no HAN ports
